### PR TITLE
Basic Toast + routes fix

### PR DESCRIPTION
**Pull request by brunellaa**
Added basic Toast component and temporary route fix.

### CODE REVIEW

File: src/pages/Overview/OverviewPage.tsx [hunk 1]
```diff
@@ -58,6 +61,10 @@ const OverviewPage = () => {
     state.user.orgs?.filter((org) => org.name === currentOrganization) ?? [];
   return (
     <OverviewContainer>
+      <Toast
+        message={"Welcome to " + currentOrganization + " 's organization panel"}
+        kind="default"
+      />
```
Comment (brunellaa) [inline src/pages/Overview/OverviewPage.tsx:67]: Toast Example,  kind="error" works too.

File: src/components/Layout.tsx [hunk 2]
```diff
@@ -38,17 +47,26 @@ const Layout = () => {
           <Header />
           <ActiveSection>
             <Switch>
-              <Route path="/notfound" component={NotFoundPage} />
+              {state.user.sessionInfo?.username ? (
+                <Route exact path="/">
+                  <Redirect to={"/" + state.user.sessionInfo?.username} />
+                </Route>
+              ) : null}
```
Comment (brunellaa) [inline src/components/Layout.tsx:54]: defaults "/" to "/username"

**Review by carsonfarmer — APPROVED** [review]
Cool, is it possible to discard the toast or otherwise have it go away?

**Comment by brunellaa** [comment]
> Cool, is it possible to discard the toast or otherwise have it go away?

it goes away after 3s, Ill add a close [x] next and custom time for fadeout

**Pull request merged**